### DISCUSSION TITLE
`ReferenceError: exports is not defined` when deploying a site with the plugin

### DISCUSSION THREAD

**Discussion by pepopowitz** [PaloAltoNetworks/docusaurus-openapi-docs#520] [Q&A]
I can't seem to get the plugin to work in the built version of my site. It works great locally, but when I build my docusaurus site, any pages generated by the plugin fail to render the right sidebar on the client, with the message `ReferenceError: exports is not defined`.

I suspect this is something specific to do with my site, but I can't figure out what. I've tried eliminating everything that I think makes it significantly different from a site freshly generated from the template, and I still can't isolate the problem.

Maybe someone has run into this, or sees something obvious in my [package.json](https://github.com/camunda/camunda-platform-docs/blob/ce2c1a41a66939acd5d838187b7c1afd4e56a7c2/package.json) or [docusaurus.config](https://github.com/camunda/camunda-platform-docs/blob/ce2c1a41a66939acd5d838187b7c1afd4e56a7c2/docusaurus.config.js)?

* **[The built site](https://641cb5af22436e00b7a92228--scintillating-mochi-243f02.netlify.app/docs/next/api/operate/add-pet/)**
* It's currently using docusaurus v2.2.0 with docusaurus-openapi-docs v1.6.1
* **[A PR that represents everything I did to install the plugin in my existing site](https://github.com/camunda/camunda-platform-docs/pull/1830)**
* **What the built site looks like in a browser**:

   <img width="1691" alt="image" src="https://user-images.githubusercontent.com/1627089/227354826-47c66df4-0889-4893-b0c6-26225922da95.png">

**Comment by sserrata** [comment]
Hi @pepopowitz, sorry to hear you're having issues. Unfortunately, I don't have time to deep dive on your issue but, if you haven't already, I recommend reaching out via the Docusaurus Discord community. If you discover the issue may stem from a bug in our plugin please let us know. Thanks and best of luck!
